View/DataContexts/SearchDataContext.cs

```diff
@@ -28,6 +29,10 @@ public static SearchDataContext Instance
         
         public List<Playlist> SearchPlaylists => PlaylistController.Create(DatabaseContext.Instance).SearchPlayListOnString(SearchTerms, UserController.CurrentUser.ID);
 
+        public List<AlbumInfo> SearchAlbumInfos =>
```

Comment:
Waarom hebben we al die search functions niet gewoon in 1?

Reply:
verschillende listen